apollo-normalized-cache-api/src/commonMain/kotlin/com/apollographql/apollo/cache/normalized/RecordFieldJsonAdapter.kt

```diff
@@ -56,37 +36,11 @@ class RecordFieldJsonAdapter {
     }
   }
 
-  @Throws(IOException::class)
-  fun from(jsonFieldSource: String): Map<String, Any?>? {
-    return fromBufferSource(Buffer().write(jsonFieldSource.encodeUtf8()))
-  }
-
-  companion object {
-
-    @Suppress("UNCHECKED_CAST")
-    @Throws(IOException::class)
-    private fun writeJsonValue(value: Any?, jsonWriter: JsonWriter) {
-      when (value) {
-        null -> jsonWriter.nullValue()
-        is String -> jsonWriter.value(value)
-        is Boolean -> jsonWriter.value(value)
-        is Number -> jsonWriter.value(value)
-        is CacheReference -> jsonWriter.value(value.serialize())
```

Comment:
Wait. Where did that logic go ? I think we still need to handle CacheReference serialization here?

Reply:
agh......jeeez my bad